### PR TITLE
Feature: volume

### PR DESCRIPTION
**Pull request by jamesjwarren**
This PR integrates volume and mute resources.

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -33,6 +33,7 @@
         <script src="components/angular-resource/angular-resource.js"></script>
         <script src="components/angular-spotify/src/angular-spotify.js"></script>
         <script src="components/angular-socket-io/socket.js"></script>
+        <script src="components/angular-debounce/dist/angular-debounce.js"></script>
```
Comment (eopa) [inline app/index.html]: angular already has a native debounce support: https://docs.angularjs.org/api/ng/directive/ngModelOptions

File: app/js/fm-player/config.js [hunk 2]
```diff
@@ -28,6 +28,12 @@ angular.module("sn.fm.player").config([
                     }],
                     currentTrack: ["PlayerTransportResource", function (PlayerTransportResource){
                         return PlayerTransportResource.get();
+                    }],
+                    PlayerVolumeResource: ["PlayerVolumeResource", function (PlayerVolumeResource){
+                        return PlayerVolumeResource.get();
+                    }],
+                    PlayerMuteResource: ["PlayerMuteResource", function (PlayerMuteResource){
+                        return PlayerMuteResource.get();
                     }]
```
Comment (eopa) [inline app/js/fm-player/config.js]: Do we need to GET this data initially? or only when we want to make changes?

File: app/index.html (hunk 1 above)
Comment (jamesjwarren) [inline app/index.html]: I tried that but it didn't seem to work with the slider as it debounces the update of ng-model which the slider relies on to adjust the display of the slider handler. We won't need this anyway if we're moving to buttons.

File: app/js/fm-player/config.js (hunk 2 above)
Comment (jamesjwarren) [inline app/js/fm-player/config.js]: Originally I was getting the volume data so we could set the position of the slider, with the buttons I don't think we would need it. Although we may still need to get the mute state so we can display it.

File: scripts.json [hunk 3]
```diff
@@ -7,15 +7,18 @@
         "app/components/angular-material/angular-material.js",
         "app/components/angular-resource/angular-resource.js",
         "app/components/angular-spotify/src/angular-spotify.js",
-        "app/components/angular-socket-io/socket.js"
+        "app/components/angular-socket-io/socket.js",
+        "app/components/angular-debounce/dist/angular-debounce.js"
     ],
```
Comment (eopa) [inline scripts.json]: are we still using debounce?

Comment (jamesjwarren) [inline scripts.json]: no, must have missed that one!

**Pull request merged**